server/utils/mappingUtils.ts

```diff
@@ -274,11 +274,11 @@ function nextCourtAppearanceToCourtAppearance(nextCourtAppearance: NextCourtAppe
     )
   }
   return {
-    nextHearingSelect: nextCourtAppearance !== undefined,
+    nextHearingSelect: nextCourtAppearance !== undefined && nextCourtAppearance !== null,
     nextHearingCourtCode: nextCourtAppearance?.courtCode,
     nextHearingType: nextCourtAppearance?.appearanceType,
     nextHearingTimeSet: typeof nextCourtAppearance?.appearanceTime === 'string',
     nextHearingDate,
-    nextCourtAppearanceAccepted: nextCourtAppearance !== undefined,
+    nextCourtAppearanceAccepted: nextCourtAppearance !== undefined && nextCourtAppearance !== null,
```

Comment:
Not going to block the PR over this one. But you can use the double bang operator `!!` to save having to check for both `null` and `undefined`:
```
let myVar = undefined;
!!myVar (returns false)

myVar = null
!!myVar (returns false)

myVar = 3
!!myVar  (returns true)

myVar = 'g'
!!myVar  (returns true)

myVar = {}
!!myVar  (returns true)
```

So this could be simplified to:
```
return {
  nextHearingSelect: !!nextCourtAppearance,
  nextHearingCourtCode: nextCourtAppearance?.courtCode,
  nextHearingType: nextCourtAppearance?.appearanceType,
  nextHearingTimeSet: typeof nextCourtAppearance.appearanceTime === 'string',
  nextHearingDate,
  nextCourtAppearanceAccepted: !!nextCourtAppearance,
}
```

Reply:
Thank you! That does look better, I'll make the change